src/qibo/engines/matrices.py

```diff
@@ -0,0 +1,139 @@
+import numpy as np
+
+
+class Matrices:
```

Comment:
Is there any special reason to use a class instead of single functions for each matrix?